src/TwigYard/Component/Application.php

```diff
@@ -109,22 +75,22 @@ private function getQueue()
             $appState,
             $mailerFactory,
             $tplFactory,
-            $this->getSiteLoggerFactory()
+            $this->getSiteLoggerFactory($globalParameters)
         );
 
         $globalLoggerFactory = $this->getGlobalLoggerFactory($globalParameters);
 
         $queue[] = new ErrorMiddleware(
             $appState,
-            $this->showErrors,
+            $globalParameters['show_errors'],
             $globalLoggerFactory,
             $this->config->getTemplateDir(),
             $this->config->getError404PageName(),
             $this->config->getError500PageName()
         );
         $queue[] = new UrlMiddleware(
             $appState,
-            $this->getConfigCache($this->cacheEnabled, $globalParameters),
+            $this->getConfigCache($globalParameters['cache_enabled'], $globalParameters),
```

Comment:
This rather ugly. We shoul be able to only pass in the array `$globalParameters`